cmd/prometheus/main.go

```diff
@@ -1384,6 +1385,11 @@ func reloadConfig(filename string, expandExternalLabels, enableExemplarStorage b
 		return fmt.Errorf("one or more errors occurred while applying the new configuration (--config.file=%q)", filename)
 	}
 
+	oldGoGC := debug.SetGCPercent(conf.GlobalConfig.GoGC)
```

Comment:
we'll need to update how we retrieve it here https://github.com/prometheus/prometheus/blob/efbd6e41c59ec8d6b7a0791c1fb337fdac53b4f2/web/web.go#L721 I think.

Reply:
Oh, I had an earlier PR that set the env var live to solve that view. I'll put that back in.